### ISSUE TITLE
ng-select not showing selected item label after writing in textbox

### ISSUE THREAD

**Issue by leonardofmed** [ng-select/ng-select#1570]
**Describe the bug**
While I'm just selecting the items from the `ng-select` element everything is working fine, but when I type something in the search box nothing is show anymore. The value is correctly selected, but the user can't understand that the element was already selected.

I did some investigation and found that when I type something and select an item, the class of `ng-select` changes. A new element is added: `ng-select-filtered`, which hide the label of the element.

I tried to create a reproduction of the error, but it seems that this is only happening in my device.

Can you tell me why this is happening and how can I fix this? Thank you very much, awesome component.

**Expected behavior**
When a item is selected, it's label is supposed to show in the search box.

Using Ionic Framework, testing in device additional information:
```
Ionic CLI                     : 5.4.16 
Ionic Framework               : @ionic/angular 4.11.7
@angular-devkit/build-angular : 0.801.2
@angular-devkit/schematics    : 8.1.3
@angular/cli                  : 8.1.3
@ionic/angular-toolkit        : 2.0.0
Cordova CLI                   : 9.0.0 (cordova-lib@9.0.1)
Android                       : 5
```

**Comment by varnastadeus** [comment]
This is default behavior

**Comment by leonardofmed** [comment]
I'm sorry, but the default behavior shows the item label when an item is selected. The behavior I'm describing is exactly the opposite. Perhaps I expressed it in a confused way. What happens is that AFTER writing in the search box and selecting the item, it does not appear in the search box. What normally happens if I don't type anything, just select the elements.

**Comment by leonardofmed** [comment]
The fix is in [#1451](https://github.com/ng-select/ng-select/issues/1451):
`<ng-select #select (close)="select.blur()"></ng-select>`
